lib/events.js

```diff
@@ -961,7 +961,7 @@ async function once(emitter, name, options = kEmptyObject) {
   const signal = options?.signal;
   validateAbortSignal(signal, 'options.signal');
   if (signal?.aborted)
```

Comment:
(non-blocking)
```suggestion
  if (signal.aborted)
```
Because we already did `validateAbortSignal`

Reply:
`validateAbortSignal` accepts `undefined` as a valid value